roles/docker/tasks/post-install.yml

```diff
@@ -5,6 +5,10 @@
     state: started
     enabled: yes
 
+- name: Find out which username is being used to log in
+  set_fact:
+    username: "{{ ansible_env.LOGNAME }}"
+
 - name: Add {{ username }} user to docker group
```

Comment:
Why not just use `ansible_env.LOGNAME` directly in this task and skip the `set_fact` above?